sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/SelectionMarkState.java

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.documentanalysis.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/** Defines values for SelectionMarkState. */
+public final class SelectionMarkState extends ExpandableStringEnum<SelectionMarkState> {
```

Comment:
Having two public models with the same name in the same library is not ideal. IntelliSense will always suggest two models and users will have to look at the package to disambiguate. But I guess this allows the 3.x models to evolve while keeping 2.x models unchanged. 

We should at least consider changing the names of the old models.


It would be good to get @JonathanGiles thoughts on this.

Reply:
That's a good question. My initial reaction is to say "no way - that's massively breaking for users", and I tend to stick on that side of the fence, even with the details we've discussed recently. We shouldn't make it harder for developers to upgrade to the new release, even if they choose to use the old APIs. At least this way, we should expect more developers to upgrade to the new version, and at some stage we can choose to add deprecation annotations to the old API and start encouraging developers to move. We should think about when deprecation should get added.